sql/testdata/select

```diff
@@ -66,7 +66,7 @@ statement ok
 INSERT INTO abc VALUES (0, 1, 2)
 
 query II
-SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b/a > 1.5 ELSE false END
+SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b::float/a::float > 1.5 ELSE false END
```

Comment:
I prefer `int / int -> float`. Truncating the results of division to an integer is understandable only because we're used to it from (some) other languages; it's surprising the first time you run across it and I think it's best to regard it as a historical accident. Also, while postgresql and sqlite truncate integer division, mysql does not, so either decision will break compatibility with *something*.